firmware/application/apps/ui_mictx.cpp.1

```diff
@@ -0,0 +1,526 @@
+/*
```

Comment:
Git saves all versions, there is no point of making local backups like this

Reply:
Thanks, I used those files as a reference when I reorganized the code and made sure it matched the newest version. Forgot to take them out.